microsoft-dynamics-365/2.5/modules/ROOT/pages/microsoft-dynamics-365-connector-reference.adoc

```diff
@@ -0,0 +1,935 @@
+= Microsoft Dynamics 365 Connector 2.4 Reference - Mule 4
+:page-aliases: connectors::microsoft/microsoft-dynamics-365-connector-reference.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Release Notes: xref:release-notes::connector/microsoft-dynamics-365-connector-release-notes-mule-4.adoc[Microsoft Dynamics 365 Connector Release Notes]
+
+== Configurations
+---
+[[dynamics-config]]
+=== Dynamics 365
+
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| * <<dynamics-config_connection, Connection>>
+* <<dynamics-config_oauth-user-pass, OAuth Username Password>>
+ | The connection types that can be provided to this configuration. | | x
+| Expiration Policy a| <<ExpirationPolicy>> | Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. This does not mean that the instance expires at the exact moment that it becomes eligible. Mule purges the instances as appropriate. | |
+| Encoding a| | The default body encoding type. If the encoding type is not specified at the operation level, this encoding type is used. | |
+|===
+
+==== Connection Types
```

Comment:
In this release, we will add support for OAuth client credentials connection as well. I think it should be described here.